packages/backend/index.js

```diff
@@ -88,6 +81,67 @@ app.get("/flocks/:code/basket", (req, res) => {
 	res.send(`Options of flock ${req.params.code}`);
 });
 
+app.post("/flocks/:coopName/:chick/vote", async (req, res) => {
+	const coopName = req.params.coopName;
+	const chickName = req.params.chick;
+	const egg = req.body.egg;
+
+	// check if flock and chick exists
+	const flock = await findFlockByCode(coopName);
+	const chick = flock.chicks.find((chick) => chick.name === chickName);
+	if (!flock) {
+		res.status(404).send({ message: "Flock not found" });
+		return;
+	}
+	if (!chick) {
+		res.status(404).send({ message: "Chick not found" });
+		return;
+	}
+
+	// handle the incoming vote
+	let voteStatus;
+	if (!egg) {
+		voteStatus = "no vote";
```

Comment:
make voteStatus an Enum probably, just using strings is a little messy/can make comparison hard